### ISSUE TITLE
Some Tags are Missing

### ISSUE THREAD

**Issue by hofq** [FTBTeam/FTB-Modpack-Issues#6235]
### Modpack

FTB Evolution

### Modpack version

1.3.0

### Launcher

FTB App

### Has the pack been modified

No

### Describe the bug

The Crafting Table Recipe is Replaced by this
![grafik](https://github.com/user-attachments/assets/c755ecd0-a7b9-45ba-8687-456451af6e7b)
![grafik](https://github.com/user-attachments/assets/def0fc0e-cb92-476a-8e2c-64f127d61a66)

### Steps to reproduce

1. farm wood
2. try craft a crafting table
3. does not work

### Expected behaviour

I can craft a craftingtable and see the correct recipe in JEI

### Screenshots

See above

**Comment by hofq** [comment]
Everything else seems to work

**Comment by Gaz492** [comment]
Would you be able to upload the logs from the modpack so we can have a look at them?
A guide on finding the logs can be found at: https://docs.feed-the-beast.com/docs/app/Instances/instance-logs

**Comment by hofq** [comment]
Oh sorry, forgot to put in that we play on a server.
I can provide you the stdout of my MC-Container tho:
[minecraft-minecraft.txt](https://github.com/user-attachments/files/17871485/minecraft-minecraft.txt)

let me know if you need more/ other logs

**Comment by hofq** [comment]
So we fixed the first Issue. On our Host the tags.js file was missing. After Reinstalling this Recipe was fine, however other broke:

We can't craft:
- Steel Rods by Modern Industrialization
![grafik](https://github.com/user-attachments/assets/4ba57cdd-0999-4fe9-b122-bc55f47b39f6)

- Process Ores by original Mod Providers where a FTB Ore is Present. I suspect that there are also Issues with the Tag.


The Difference now is that we replaced the whole file. I even downloaded it seperately and placed it in server folder and re-installed the server files 2 times.


Is there a specific file i could try to replace to fix it?

**Comment by ArtpokeBr** [comment]
Yeah this is probably due to Loading Order Issues, We've been dealing with a bunch of them recently, in the next update hopefully all of this will be fixed ✋
